src/utils/utils.ts

```diff
@@ -26,16 +26,43 @@ export const geoToTopo = (features, sphericalArea) => {
   return topo;
 };
 
-// 2022-03-15 => March 15, 2022
-export const prettyDate = (date: string): string => {
+export const getMonth = (date: string): string => {
+  const dateParts = date.split("-").map((p) => parseInt(p));
+  return new Date(
+    dateParts[0],
+    dateParts[1] - 1, // months are 0-indexed in js
+    dateParts[2]
+  ).toLocaleDateString("en-US", {
+    month: "long",
+  });
+};
```

Comment:
Okay, so I think there's a really much cleaner way to do this. I just tried this on my local node console, so let me know if this doesn't work for you? (I also am going to change this to a function as opposed to an anonymous function saved to a variable because theres some weird stuff that happens if you do that so I've been trying to get into the habit.)

```suggestion
// 2022-03-15 => "March"
export function getMonth(date: string): string {
  return new Date(date).toLocaleDateString("en-US", {
    month: "long",
  });
};
```